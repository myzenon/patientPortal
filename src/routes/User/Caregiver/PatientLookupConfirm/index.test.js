import React from 'react'
import { CaregiverPatientLookupConfirm } from './index'
import ReactGA from 'react-ga';

   // This should be part of your setup
   ReactGA.initialize('UA-121783543-1', {
       'debug': true
     });;

jest.mock("react-ga")

test('renders correctly', () => {
    const props = {
        page: {
            apiData: {
                firstName: 'firstName',
                lastName: 'lastName'
            }
        }
    }
    const wrapper = render (
        <CaregiverPatientLookupConfirm {...props} />
    )
    expect(toJson(wrapper)).toMatchSnapshot()
})