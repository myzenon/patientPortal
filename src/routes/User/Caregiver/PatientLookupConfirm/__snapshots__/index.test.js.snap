// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="container-fluid"
>
  <div
    class="content container"
  >
    <p
      class="instruction"
    >
      Awesome! It looks like you are trying to help 
      <strong>
        firstName lastName
      </strong>
       today.
    </p>
    <p
      class="instruction"
    >
      Is that right?
    </p>
    <div
      class="choicesWrapper"
    >
      <div
        class="row"
      >
        <div
          class="col"
        >
          <a
            href="/caregiver-consent"
          >
            <button
              class="button btn btn-secondary"
            >
              YES
            </button>
          </a>
        </div>
        <div
          class="col"
        >
          <a
            href="/caregiver-patient-lookup"
          >
            <button
              class="button btn btn-secondary"
            >
              NO
            </button>
          </a>
        </div>
      </div>
    </div>
    <p
      class="hint"
    >
      As a caregiver, you will be asked questions as if YOU were the patient. 
      <br />
      We've put you in their shoes!
    </p>
  </div>
  <div
    class="footerWrapper container"
  >
    <div>
      <div
        class="wrapper"
      >
        <div
          class="stepActive"
        >
          Account
          <br />
          Setup
          <div
            class="stepNumberActive"
          >
            1
          </div>
        </div>
        <div
          class="step"
        >
          Medication
          <br />
          Review
          <div
            class="stepNumber"
          >
            2
          </div>
        </div>
        <div
          class="step"
        >
          Additional 
          <br />
          Medications
          <div
            class="stepNumber"
          >
            3
          </div>
        </div>
        <div
          class="step"
        >
          Health
          <br />
          Updates
          <div
            class="stepNumber"
          >
            4
          </div>
        </div>
        <div
          class="step"
        >
          Set
          <br />
          Appointment
          <div
            class="stepNumber"
          >
            5
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
